lntest/btcd.go

```diff
@@ -76,7 +76,7 @@ func NewBackend(miner string) (*BtcdBackendConfig, func(), error) {
 		"--logdir=" + logDir,
 		"--connect=" + miner,
 	}
-	netParams := &chaincfg.SimNetParams
+	netParams := &chaincfg.RegressionNetParams
```

Comment:
Nit: would be nice if this was provided as a function parameter.

Reply:
Done.